app/src/main/res/values/strings.xml

```diff
@@ -183,6 +183,8 @@
     <string name="albums">Albums</string>
     <string name="artists">Artists</string>
     <string name="yes">Yes</string>
+    <string name="confirm_delete_all_downloads">Are you sure to delete all your downloaded files? </string>
```

Comment:
By reading this line it seems like the files would be cleared. Is that the case?
```suggestion
    <string name="confirm_delete_all_downloads">Are you you want sure to delete all your downloaded files?</string>
```